lib/cartopy/mpl/ticker.py

```diff
@@ -198,13 +326,25 @@ def __init__(self,
             The symbol used to represent degrees. Defaults to u'\u00B0'
             which is the unicode degree symbol.
         number_format: optional
-            Format string to represent the longitude values. Defaults to
-            'g'.
+            Format string to represent the degrees tick values.
```

Comment:
I find this wording a bit confusing, from a user's perspective.  I prefer `Format string to represent the longitude values`.

Reply:
I reformulated the description of both  `number_format` and `seconds_number_format`.